src/server/game/Spells/Spell.cpp

```diff
@@ -4727,7 +4727,7 @@ SpellCastResult Spell::CheckCast(bool strict)
                 return SPELL_FAILED_NOT_READY;
         }
 
-        if (!m_caster->GetSpellHistory()->IsReady(m_spellInfo))
+        if (!m_caster->GetSpellHistory()->IsReady(m_spellInfo) && !(_triggeredCastFlags & TRIGGERED_IGNORE_SPELL_AND_CATEGORY_CD))
```

Comment:
Can't decide what is lesser evil, either removing the cooldown of spell 178368 before casting the spell or removing it from the SpellCategory cooldowns. Do you have any suggestions or a third less hacky way how to make it work?